_includes/sub-template-dnszone.md

```diff
@@ -0,0 +1,37 @@
+
+<!--
+Copyright (c) 2021, Oracle and/or its affiliates.
+Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+-->
+
+### Custom DNS (optional)
+
+#### Registered Domain Name
+
+You need to buy a domain name to create a custom DNS alias.
+
+#### Azure DNS Zone
+
+If you create the DNS alias on an existing [Azure DNS Zone](https://docs.microsoft.com/en-us/azure/dns/dns-overview), make sure you have perfomed the [Azure DNS Delegation](https://docs.microsoft.com/en-us/azure/dns/dns-domain-delegation).  Once you have completed the delegation, you can verify it with `nslookup`.  For example, assuming your domain name is **contoso.com**, this output shows a correct delegation.
```

Comment:
Once you have completed ->  After you have completed

Reply:
Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb